ert_shared/ensemble_evaluator/entity/snapshot.py

```diff
@@ -190,6 +190,13 @@ def to_dict(self):
     def get_status(self):
         return self._data["status"]
 
+    def is_finished(self):
+        for real in self._data["reals"].values():
```

Comment:
This ignores will count inactive reals as well. You need to check if they are active.